src/odemis/gui/cont/tabs.py

```diff
@@ -1824,15 +1824,28 @@ def __init__(self, name, button, panel, main_frame, main_data):
             calib_prefix="calib_2",
         )
 
+        if model.getMicroscope().name.lower().endswith("sim"):  # it's a simulator
```

Comment:
The GUI already has a pointer to the microscope, let's use it:
```suggestion
        if main_data.microscope.name.lower().endswith("sim"):  # it's a simulator
```